packages/testlab/src/http-error-logger.ts

```diff
@@ -0,0 +1,24 @@
+import {Request} from 'express';
+import {IncomingMessage} from 'http';
+
+export function createUnexpectedHttpErrorLogger(
```

Comment:
Please add API documentation explaining how to use this helper and more importantly, describe this new helper in testlab's README: https://github.com/strongloop/loopback-next/blob/master/packages/testlab/README.md